factory/factory.go

```diff
@@ -21,20 +22,22 @@ var AusfConfig Config
 
 // TODO: Support configuration update from REST api
 func InitConfigFactory(f string) error {
-	if content, err := os.ReadFile(f); err != nil {
+	content, err := os.ReadFile(f)
+	if err != nil {
 		return err
-	} else {
-		AusfConfig = Config{}
-
-		if yamlErr := yaml.Unmarshal(content, &AusfConfig); yamlErr != nil {
-			return yamlErr
-		}
-		if AusfConfig.Configuration.WebuiUri == "" {
-			AusfConfig.Configuration.WebuiUri = "webui:9876"
-		}
 	}
+	AusfConfig = Config{}
 
-	return nil
+	if yamlErr := yaml.Unmarshal(content, &AusfConfig); yamlErr != nil {
+		return yamlErr
```

Comment:
```suggestion
	if err = yaml.Unmarshal(content, &AusfConfig); err != nil {
		return err
```

There is no need for a new error variable here.